File: Java Spicy-snack/Calculator.java
import java.util.Scanner;
public class Calculator{
    public static void main(String[]args){
    Scanner input = new Scanner(System.in);
    
    System.out.println("Enter first integer: ");
    int firstInterger = input.nextInt();

    System.out.println("Enter arithmetric operator: ");
    String arithmeticOperator = input.next();

    System.out.println("Enter second integer: ");
    int secondInterger = input.nextInt();

    int sum = firstInterger + secondInterger;
    int subtraction = firstInterger - secondInterger;
    int product = firstInterger * secondInterger;
    double division = firstInterger / secondInterger;

    
    if(arithmeticOperator.equals("+")){
      System.out.println(sum);
    }

    if(arithmeticOperator.equals("-")){
      System.out.println(subtraction);
    }

    if(arithmeticOperator.equals("*")){
      System.out.println(product);
    }
    if(arithmeticOperator.equals("/")){
      System.out.println(division);
    }

    }
}

File: Spicy-Snack/calculator.py

first_user_input = int(input("Enter first number: "))
arithmetic_operator = input("Enter an arithmetic sign: ")
second_user_input = int(input("Enter second number: "))
  

addition= first_user_input + second_user_input;
subtraction = first_user_input - second_user_input;
product = first_user_input * second_user_input;
division = first_user_input // second_user_input;

    
if(arithmetic_operator == '+'):
      print(addition)
if(arithmetic_operator == '-'):
      print(subtraction)
if(arithmetic_operator == '*'):
      print(product);
if(arithmetic_operator == '//'):
      print(division);

